ext/cfx-ui/src/app/servers/components/detail/servers-detail.component.html

```diff
@@ -44,6 +44,8 @@
 								[class.ag]="server.premium === 'ag'"
 								[class.au]="server.premium === 'au'"
 								[class.pt]="server.premium === 'pt'"
+								data-balloon-pos="right"
```

Comment:
This tooltip will only show if there's a premium value already, which at this point will only be ag/au/pt.